electrum/commands.py

```diff
@@ -1162,6 +1162,7 @@ def add_global_options(parser):
     group.add_argument("--regtest", action="store_true", dest="regtest", default=False, help="Use Regtest")
     group.add_argument("--simnet", action="store_true", dest="simnet", default=False, help="Use Simnet")
     group.add_argument("-o", "--offline", action="store_true", dest="offline", default=False, help="Run offline")
+    group.add_argument("--btcv_regtest", action="store_true", dest="btcv_regtest", default=False, help="Use BTCV Regtest")
```

Comment:
Since electrum-vault is a wallet only for BTCV, the `--regtest` call should be used to enable BTCV regrets.

Reply:
done